posthog/models/feature_flag/flag_analytics.py

```diff
@@ -62,6 +64,19 @@ def _extract_total_count_for_key_from_redis_hash(client: redis.Redis, key: str)
     return total_count, min_time, max_time
 
 
+def capture_usage_for_all_teams(ph_client: "Posthog") -> None:
+    for team in (
+        Team.objects.select_related("organization")
+        .exclude(Q(organization__for_internal_metrics=True) | Q(is_demo=True))
+        .only(
+            "id",
+            "uuid",
+            "organization__for_internal_metrics",
```

Comment:
You don't need to include this here in order for it to work in the `exclude`. I'm assuming you don't use it elsewhere, since it's excluded from the Teams object :)

Reply:
hmm, pretty sure I do need it when not selecting any other org field. Leads to this error:

> django.core.exceptions.FieldError: Field Team.organization cannot be both deferred and traversed using select_related at the same time

Reply:
Oh do you need the org then? It looks like you just pass in the team attributes here:

`capture_team_decide_usage(ph_client, team.id, team.uuid)`

Reply:
oh yes indeed, you're right 🤦 , the exclude doesn't need it to be selected, nice thanks!